packages/client/test/cli/cli.spec.ts

```diff
@@ -97,17 +401,281 @@ describe('[CLI]', () => {
+          assert.fail(`child process exited with code ${code}`)
+        }
+      })
+    })
+  }, 18000)
+  it('should start client with custom input for storage cache size', async () => {
+    const file = require.resolve('../../dist/bin/cli.js')
+    const cliArgs = ['--storageCache=2000']
+    const child = spawn(process.execPath, [file, ...cliArgs])
+    return new Promise((resolve) => {
+      child.stdout.on('data', async (data) => {
+        const message: string = data.toString()
+        if (message.includes('storage cache')) {
+          assert.ok(message.includes('2000'), 'storage cache option works')
+          child.kill()
+          resolve(undefined)
+        }
+        if (message.toLowerCase().includes('error')) {
+          child.kill(9)
+          assert.fail(`client encountered error: ${message}`)
+        }
+      })
+      child.stderr.on('data', (data) => {
+        const message: string = data.toString()
+        assert.fail(`stderr: ${message}`)
+      })
+      child.on('close', (code) => {
+        if (typeof code === 'number' && code > 0) {
+          assert.fail(`child process exited with code ${code}`)
+        }
+      })
+    })
+  }, 18000)
+  it('should start client with custom input for trie cache size', async () => {
+    const file = require.resolve('../../dist/bin/cli.js')
+    const cliArgs = ['--trieCache=2000']
+    const child = spawn(process.execPath, [file, ...cliArgs])
+    return new Promise((resolve) => {
+      child.stdout.on('data', async (data) => {
+        const message: string = data.toString()
+        if (message.includes('trie cache')) {
+          assert.ok(message.includes('2000'), 'trie cache option works')
+          child.kill()
+          resolve(undefined)
+        }
+        if (message.toLowerCase().includes('error')) {
+          child.kill(9)
+          assert.fail(`client encountered error: ${message}`)
+        }
+      })
+      child.stderr.on('data', (data) => {
+        const message: string = data.toString()
+        assert.fail(`stderr: ${message}`)
+      })
+      child.on('close', (code) => {
+        if (typeof code === 'number' && code > 0) {
+          assert.fail(`child process exited with code ${code}`)
+        }
+      })
+    })
+  }, 18000)
+  // test experimental feature options
+  it('should start client when passed options for experimental features', async () => {
+    const file = require.resolve('../../dist/bin/cli.js')
+    const cliArgs = ['--mine=true', '--forceSnapSync=true', '--dev=poa', '--port=30304']
+    const child = spawn(process.execPath, [file, ...cliArgs])
+    return new Promise((resolve) => {
+      child.stdout.on('data', async (data) => {
+        const message: string = data.toString()
+        if (message.includes('Client started successfully')) {
+          assert.ok(
+            message.includes('Client started successfully'),
+            'Clients started with experimental feature options'
```

Comment:
If there's a way to infer that snap sync did start (from the logs), it would be nice to capture that here.

Reply:
Currently, the messages still aren't updated to log that snap sync is happening. Will change that and update the tests in a different PR.